lti_consumer/tests/unit/test_lti_xblock.py

```diff
@@ -132,14 +133,32 @@ def test_context_id(self):
         """
         self.assertEqual(self.xblock.context_id, str(self.xblock.course_id))
 
-    def test_validate(self):
+    @ddt.data(['x=y'], ['  x  =  y  '], ['x=${y}'], ['  x  =  ${y} '], ['x=${  y  }'])
```

Comment:
@Jacatove Sure, let me add a larger list of scenarios, the pattern can be tested here: https://regex101.com/ 'x=$500' is not valid since $ is nor part of the set of characters of w [A-Za-z0-9_].